.github/workflows/static.yml

```diff
@@ -0,0 +1,13 @@
+name: Static analysis
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+  schedule:
+    - cron: '0 0 * * 1'
+
+jobs:
+  static:
+    uses: dvsa/.github/.github/workflows/php-static.yml@v3.2.0
```

Comment:
**New file**

Static code analysis Github action

Reply:
We need to reference a specific tag on the `.github` repo, and there isn't a `v3`